package.json

```
@@ -65,8 +66,9 @@
     "cross-env": "^5.2.1",
     "dotenv": "^16.0.3",
     "eslint": "^8.57.1",
-		"eslint-import-resolver-webpack": "^0.13.8",
-    "eslint-plugin-jest": "^26.9.0",
+    "eslint-import-resolver-webpack": "^0.13.10",
+		"eslint-plugin-jsdoc": "^46.10.1",
```

The whitespace here looks wrong.
```suggestion
    "eslint-plugin-jsdoc": "^46.10.1",
```